PM2 - REST service - second PR

Mchapuis opened the pull request:
This is the original branch for the REST service. This branch is only for PM2 part 3.

Commit:
REST files pre-linking to web service

Commit:
Generator fixed, GeneratorWS fixed, Rest almost complete. Temp commit in order to bring Generator and GeneratorWS in master, will then continue Rest in this current branch (PM2-Post-Soap-Fixes)

Commit:
Forgot in previous commit. Modified to add is GUI to not have progress dialog when running from web service

Commit:
Pre final test for REST

Commit:
Finished REST.

REST Wrapper for the Generator
Java Client to make REST request to the generator
Javascript client to make REST requests to the generator

jashanj0tsingh commented:
@Mchapuis Thank you for the second PR, however we are still at square one. You will need to resolve the conflicts at some point before I can merge them after review. @smokhov

smokhov commented:
@Mchapuis -- the conflicts listed can be resolved online here on GitHub by editing them out; err on the side of inclusion.

Mchapuis committed:
Revert "Merge branch 'WS' into PM2-Post-Soap-Fixes"

This reverts commit e379f82d03db4323343681796fb1683eea876223.

Mchapuis commented:
I fixed the conflicts. I still need to update the README for this service tomorrow.

smokhov commented:
@Mchapuis -- are we still waiting for the README here?

Mchapuis committed:
Update of ReadMe for REST

Mchapuis committed:
Merge branch 'WS' into PM2-Post-Soap-Fixes

Mchapuis commented:
1- I reverted the merge from ws in case this was creating issues ( I was seeing changes in my branch )
2- I fixed the conflicts with the base branch ( similar fixes than last time )
3- Updated the README for REST

Mchapuis commented:
** I just realized that I am creating a new commit when I resolve conflicts. I can see a new commit called `Merge branch WS into PM2- Post-Soap-Fixes` .**

jashanj0tsingh reviewed (commented):
I am still unable to deploy this pull request locally. It will help if the README can be more elaborate. I tried to locally test this pull request, builds fine but won't deploy. Can you share some specs like the GlassFish version, port numbers and some similar steps/hacks that you have on yourlocal machines which can help resolve this issue?

Mchapuis commented:
The full documentation has been pushed to bitbucket. I hope this will answer your questions.
